Base/usr/share/man/man8/TelnetServer.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+TelnetServer - Serenity telnet server
+
+## Synopsis
+
+```**sh
+$ TelnetServer [options]
+```
+
+## Description
+
+`TelnetServer` is a basic telnet server for Serenity. By default, it
+runs on port 23 and provides a shell upon connection.
+
+## Options
+
+* `-p`: Choose different port for TelnetServer to attach to.
```

Comment:
So should "TelnetServer" be monospaced or not? I'd say no, but what's important is that we pick one way and stick to it :)